src/MarchingCubes.jl

```diff
@@ -64,7 +64,7 @@ struct MC{F,I}
     x::RefValue{Vector{F}}
     y::RefValue{Vector{F}}
     z::RefValue{Vector{F}}
-
+    normal_sign::Int # direction of normal vectors (+1 for outward / -1 for inward)
```

Comment:
```suggestion
    normal_sign::Int  # direction of normal vectors (+1 for outward / -1 for inward)
```

nitpicking on the number of spaces :)